components/SerialManager/SerialManager/SerialManager.cpp

```diff
@@ -42,9 +65,10 @@ void SerialManager::try_receive()
     // Notify main that a command was received during startup
     notify_startup_command_received();
 
-    const auto result = this->commandManager->executeFromJson(std::string_view(reinterpret_cast<const char *>(this->data)));
-    const auto resultMessage = result.getResult();
-    usb_serial_jtag_write_bytes(resultMessage.c_str(), resultMessage.length(), 1000 / 20);
+  const auto result = this->commandManager->executeFromJson(std::string_view(reinterpret_cast<const char *>(this->data)));
+  const auto resultMessage = result.getResult();
+  int written = usb_serial_jtag_write_bytes(resultMessage.c_str(), resultMessage.length(), 1000 / 20);
+  (void)written; // ignore errors if driver already uninstalled
```

Comment:
Do we care about the amount written here? Or is this done to satisfy some warnings?